.circleci/config.yml

```diff
@@ -53,11 +53,6 @@ jobs:
     executor: rsp-large
     steps:
       - checkout
-      - restore_cache:
```

Comment:
ah, looks like i went back on this due to size
I think we can discuss if we want to do this or not
right now, the install time on CI has stayed roughly the same (technically it's a little faster than it used to be) so that's why I left off the manual caching we were doing previously